examples/systemcalls/systemcalls.c

```diff
@@ -80,11 +178,75 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
         command[i] = va_arg(args, char *);
     }
     command[count] = NULL;
-    // this line is to avoid a compile warning before your implementation is complete
-    // and may be removed
-    command[count] = command[count];
-
+    
+    openlog(NULL, 0, LOG_USER);
+    int fd, result_dup, status;
+    pid_t pid, result_wait;
+    fd = open(outputfile, O_RDWR | O_CREAT, S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH);
```

Comment:
@darshitagrawal think about how you could combine logic with the function above.